docs/getting-started/security-ui.asciidoc

```diff
@@ -51,6 +51,20 @@ View event and host counts specific to Elastic data shippers and apps, such as *
 [role="screenshot"]
 image::images/events-count.png[]
 
+*Threat Intelligence card*
+
+The Threat Intelligence card shows the total number of threat indicator events collected by the Threat Intel Filebeat module. This module ingests, parses, and visualizes data from several open source threat feeds to provide you with relevant and timely threat intelligence. As a security analyst, you can use this data in a variety of ways. For example, you can reference this data while building new threat indicator match rules or triaging alerts. Or, you use it to assess the robustness of your organization's security controls and make suggestions for improvements. Regardless of your goal, the Threat Intel Filebeat module offers you timely and actionable threat intelligence that you can leverage for your, or your organization's, detection and prevention needs.
```

Comment:
I tried to find whether Elastic has specific guidelines on "open source" vs. "open-source," but couldn't find anything (or didn't know yet where to look). But I think general usage would be hyphenated for an adjective form like this.

```suggestion
The Threat Intelligence card shows the total number of threat indicator events collected by the Threat Intel Filebeat module. This module ingests, parses, and visualizes data from several open-source threat feeds to provide you with relevant and timely threat intelligence. As a security analyst, you can use this data in a variety of ways. For example, you can reference this data while building new threat indicator match rules or triaging alerts. Or, you use it to assess the robustness of your organization's security controls and make suggestions for improvements. Regardless of your goal, the Threat Intel Filebeat module offers you timely and actionable threat intelligence that you can leverage for your, or your organization's, detection and prevention needs.
```